src/gui/transferlistmodel.h

```diff
@@ -32,13 +32,57 @@
 
 #include <QAbstractListModel>
 #include <QList>
+#include <QWidget>
+
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
just found out that you can forward declare `enum class TorrentState;` then you won't need to make this change.

Reply:
So should I do it?

Reply:
done

Reply:
just leave it for now...

Reply:
... but it is already used through out the code base, well, make your choice.

Reply:
You marked this resolved so I am not reverting the commit.

Reply:
Sorry, I pressed that button too quickly.
I think I'll make another PR that prohibits forward declaring enum class. So it would be best that we make it right here.

Reply:
I think you meant to revert the commit, so I reverted it

Reply:
my English skills are still not that good. I'm working on it 😂😊

Reply:
>I think you meant to revert the commit, so I reverted it

yes

>my English skills are still not that good. I'm working on it 

no worries :)

Reply:
OK I did some test and I have cleared my confusion. In short, forward declaring enum class is okay since the compiler (gcc) will check if the underlying type is the same. So sorry about the misleading discussion above. You can drop the `Revert "TorrentState"` commit and you also should not use const reference instead of a plain value (see torrentcontentmodelfile.h for example).

Reply:
done.